### DISCUSSION TITLE
Analytical Beam Solution in BeamDyn

### DISCUSSION THREAD

**Discussion by Pouria-AZD** [OpenFAST/openfast#869] [Q&A]
Hello,

Does anyone here can help me to find an analytical solution for a concentrated point load at the tip in order to compare with BeamDyn results like the one for the concentrated  moment at the tip which is already exists in Qi Wang et al.

Thank you in advance.

Kind regards,
Pouria AMOUZADRAD

**Comment by ebranlard** [comment]
Hi, 
For straight beams of constant properties, you can find these results in the book: "Roark's Formulas for Stress and Strain" by Young, in the chapter about beams and bars. It's usually in the first table, labelled "Shear, moment, slope, and deflection formulas for elastic straight beams".

For the 6th edition: Ch7, Table 3, page 100. 
For the 7th edition: Ch8, Table 8.1, page 189.

**Comment by Pouria-AZD** [comment]
Dear @ebranlard ,

There is an analytical solution to calculate tip displacements with a concentrated moment at the tip here in https://www.nrel.gov/docs/fy14osti/60759.pdf at the end of page 6. 
The one that I want is to find an analytical solution for a point load at the tip for each X,Y and Z direction.

**Comment by ebranlard** [comment]
For a geometrically exact beam, I actually don't know! Maybe someone else can pitch in.
